package.json

```diff
@@ -31,15 +30,14 @@
     "repository": "finn-no/node-brakes-prometheus",
     "dependencies": {},
     "peerDependencies": {
-        "prom-client": ">=12"
+        "prom-client": ">=13"
```

Comment:
I left this at 12 because there is nothing in this library other than this line that was preventing it from working with v12. That said it is probably good to require the same version being tested against.